ui/app/components/pending-tx.js

```diff
@@ -425,14 +426,16 @@ PendingTx.prototype._notZeroOrEmptyString = function (obj) {
 
 function forwardCarrat () {
   return (
-
     h('img', {
       src: 'images/forward-carrat.svg',
       style: {
         padding: '5px 6px 0px 10px',
         height: '37px',
       },
     })
-
   )
 }
+
+function cloneObj (obj) {
+  return JSON.parse(JSON.stringify(obj))
+}
```

Comment:
lets use the `clone` module instead https://github.com/MetaMask/metamask-plugin/blob/master/package.json#L138